movie/src/components/ButtonLoadMore/index.jsx

```diff
@@ -0,0 +1,15 @@
+import "./ButtonLoadMore.css";
+const ButtonLoadMore = ({ onClick, fourIncreasement, moviesDisplay }) => {
+  return (
+    <button
+      disabled={fourIncreasement > moviesDisplay}
+      onClick={() => onClick()}
```

Comment:
you can just do onClick={onClick}. 
if we don't have to pass any params to the function, no need to write it as an arrow function